test/unit/katello/rhsm_fact_parser_test.rb

```diff
@@ -16,6 +16,14 @@ def setup
         'net.interface.eth2.permanent_mac_address' => '52:54:00:D8:27:F7',
         'net.interface.eth3.ipv4_address' => 'Unknown',
         'net.interface.eth3.mac_address' => '00:00:00:00:00:14',
+        'net.interface.eno1.ipv6_address.global' => '2006:ac81::1105',
+        'net.interface.eno1.ipv6_address.global_list' => '2006:ac81::1105',
```

Comment:
I'd suggest to use the example range: `2001:db8::/32` is defined in RFC3849. See https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600 as well.

Reply:
This is actually from reserved range, but I can change this.